packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -7,7 +7,10 @@ import 'package:url_launcher_platform_interface/url_launcher_platform_interface.
 
 import 'package:platform_detect/platform_detect.dart' show browser;
 
-const _mailtoScheme = 'mailto';
+const _safariTargetTopSchemes = {
+  'mailto',
+  'tel',
+};
```

Comment:
Can you also add `'sms'` (and a test for it)? That way we can kill two birds (issues) with one stone (this pr)! :)

Reply:
Done!

And also added some more tests that were missing for tel and added the same for sms